pkg/compression/compression_test.go

```diff
@@ -15,13 +15,12 @@ import (
 
 func TestDetectCompression(t *testing.T) {
 	cases := []struct {
-		filename      string
-		unimplemented bool
+		filename string
```

Comment:
(Non-blocking: This could become a `[]string`.)

Reply:
It could, but I prefered the `c.filename` as it's more descriptive (and makes the diff smaller). But I could switch it if you prefer.

Reply:
No, this is fine as well.